egs/swbd/s5c/local/nnet3/run_cldnn.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# Copyright 2015  Johns Hopkins University (Author: Daniel Povey).
+#           2015  Vijayaditya Peddinti
+#           2015  Xingyu Na
+#           2015  Pegah Ghahrmani
+#           2016  Tom Ko
+# Apache 2.0.
+
+
+# this is a basic lstm script
+# LSTM script runs for more epochs than the TDNN script
+# and each epoch takes twice the time
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call lstm/train.sh with --gpu false
+
+stage=9
+train_stage=-10
+has_fisher=true
+affix=
+speed_perturb=true
+common_egs_dir=
+
+# CNN options
+cnn_indexes="3,8,1,1,256,1,3,1,1,3,1"
```

Comment:
explain in detail the what these hyper-parameters are. For complicated components like I would suggest using python dict format.  e.g.
<pre><code>
cnn_parameters='{
'layer1': {
                 'filter-x-dim' : 3,
                 'filter-y-dim' : 8,
                 'filter-x-step' : 1,
                 'filter-y-step' : 1,
                 'num-filters' : 256
     }
}'

You can directly parse this string in python using eval.